src/components/Editor/index.js

```diff
@@ -1,264 +1,69 @@
-import React, { Component } from 'react'
-import cn from '../../utils/cn'
-import prism from '../../utils/prism'
-import normalizeCode from '../../utils/normalizeCode'
-import normalizeHtml from '../../utils/normalizeHtml'
-import htmlToPlain from '../../utils/htmlToPlain'
-import selectionRange from '../../vendor/selection-range'
-import { getIndent, getDeindentLevel } from '../../utils/getIndent'
+import React, { Component, Fragment } from 'react';
+import Editor from 'react-simple-code-editor';
+import Highlight, { Prism } from 'prism-react-renderer';
+import { theme as liveTheme } from '../../constants/theme';
 
-class Editor extends Component {
-  static defaultProps = {
-    contentEditable: true,
-    language: 'jsx'
-  }
-
-  undoStack = []
-  undoOffset = 0
-  undoTimestamp = 0
-  compositing = false
-
-  state = {
-    html: ''
-  }
-
-  onRef = node => {
-    this.ref = node
-  }
-
-  getPlain = () => {
-    if (this._innerHTML === this.ref.innerHTML) {
-      return this._plain
+class CodeEditor extends Component {
```

Comment:
Is there a way to prevent the content from being editable with `react-simple-code-editor`? Do we want to maintain that piece of the API or can you pass it to the editor directly? I see below the explicit prop has been scrapped but I understand it might be passable through `...rest` props that make it into the component anyway

Reply:
Resolved with the new `disabled` prop